2002-06-27  Jim Wilson  <wilson@redhat.com>

	* cp/decl.c (mark_named_label_lists): Add missing & in call to
	mark_binding_level.

--- gcc/cp/decl.c.jj	Fri Apr 12 22:16:12 2002
+++ gcc/cp/decl.c	Wed Jul  3 17:52:11 2002
@@ -2042,7 +2042,7 @@ mark_named_label_lists (labs, uses)
   for (; l; l = l->next)
     {
       ggc_mark (l);
-      mark_binding_level (l->binding_level);
+      mark_binding_level (&l->binding_level);
       ggc_mark_tree (l->old_value);
       ggc_mark_tree (l->label_decl);
       ggc_mark_tree (l->bad_decls);
